.github/workflows/run_doublet-detection.yml

```diff
@@ -24,7 +24,9 @@ on:
       - main
     paths:
       - analyses/doublet-detection/**
-      - .github/workflows/run_doublet-detection.yml
+      - "!analyses/doublet-detection/Dockerfile"
+      - "!analyses/doublet-detection/.dockerignore"
+      # - .github/workflows/run_doublet-detection.yml
```

Comment:
I think you probably want to track reverting this in an issue.

Reply:
Added this as part of #676, if that works?